rpcs3/rpcs3qt/gui_application.cpp

```diff
@@ -161,7 +161,11 @@ bool gui_application::Init()
 	{
 		welcome_dialog* welcome = new welcome_dialog(m_gui_settings, false);
 
-		welcome->exec();
+		if (welcome->exec() == QDialog::Rejected)
+		{
+			// if the agreement on RPCS3's usage conditions was not accecped by the user, ask the main window to softly terminate
```

Comment:
```suggestion
			// If the agreement on RPCS3's usage conditions was not accepted by the user, ask the main window to terminate gracefully
```

Reply:
... that after I read and read the comments before the commit